tensorboard/webapp/plugins/text_v2/data_source/text_v2_types.ts

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
I don't know if `types` is the correct name for this file. It's actually more your service interface - maybe this should just be text_data_source.ts. But I don't feel strongly about it.

Reply:
Done, thanks.